packages/api-rx/src/index.ts

```diff
@@ -0,0 +1,297 @@
+// Copyright 2017-2018 @polkadot/api-rx authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { RxApiInterface, QueryableStorageFunction, QueryableModuleStorage, QueryableStorage, SubmittableExtrinsics, SubmittableModuleExtrinsics, SubmittableExtrinsicFunction } from './types';
+
+import E3 from 'eventemitter3';
+import { EMPTY, Observable, from } from 'rxjs';
+import { defaultIfEmpty, map } from 'rxjs/operators';
+import WsProvider from '@polkadot/rpc-provider/ws';
+import RpcRx from '@polkadot/rpc-rx/index';
+import { Extrinsics, ExtrinsicFunction } from '@polkadot/extrinsics/types';
+import extrinsicsFromMeta from '@polkadot/extrinsics/fromMetadata';
+import { Storage } from '@polkadot/storage/types';
+import storageFromMeta from '@polkadot/storage/fromMetadata';
+import { Base } from '@polkadot/types/codec';
+import { Hash, Method, RuntimeVersion } from '@polkadot/types/index';
+import RuntimeMetadata from '@polkadot/types/Metadata';
```

Comment:
we could move lines 16-18 up into the types section

Reply:
It is becoming a nightmare on these - since they are types or classes. (With @polkadot/types it is a struggle to get "right" atm)